### ISSUE TITLE
ENR-EYE NTR: Cyclic esotropia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#2959]
[added] Added//Cyclic esotropia//new term request; definition: Normal binocular single vision alternates with large angle esotropia in rhythmic cycle [BIOS 2012] (PS); parent- Distance esotropia [level: 12]; term- Cyclic esotropia [level: 12]

**Comment by pnrobinson** [comment]
@psergouniotis
Is this a synonym of Alternating esotropia?

**Comment by psergouniotis** [comment]
No they are not synonyms.

Cyclic esotropia definition: Convergent squint in which normal binocular single vision is alternating with large angle esotropia in rhythmic cycle.
Alternating esotropia definition: Esotropia in which either eye may be used for fixation.

In simpler terms, cyclic suggests squint alternating with time rhythmically. 
Alternating suggests that the patient may be using either eye to fixate (which means that you do not have the same one eye squinting all the time).

**Comment by pnrobinson** [comment]
thanks! Adding definitions.
